test/fuzz/transformation_composite_construct_test.cpp

```diff
@@ -142,10 +142,15 @@ TEST(TransformationCompositeConstructTest, ConstructArrays) {
   TransformationCompositeConstruct make_vec2_array_length_3_bad(
       37, {41, 45, 27, 27}, MakeInstructionDescriptor(46, SpvOpAccessChain, 0),
       200);
+  // This test covers line 164.
```

Comment:
Bad idea to use absolute line numbers in a comment. Instead, explain conceptually what the check covers.

Also it's better to say "This assertion" or "This check", rather than "This test", since the whole function is a test.